samples/network/manage-ip-address/Program.cs

```diff
@@ -24,38 +25,33 @@ public class Program
         //   - Get the assigned public IP address for a virtual machine
         //   - Remove a public IP address from a virtual machine.
 
-        private static readonly string UserName = "tirekicker";
-        private static readonly string Password = "<password>";
-        private static readonly string SubscriptionId = Environment.GetEnvironmentVariable("AZURE_SUBSCRIPTION_ID");
 
-        public static async Task RunSample(TokenCredential credential)
+        public static async Task RunSample()
         {
-            string publicIPAddressName1 = Utilities.RandomResourceName("pip1", 20);
-            string publicIPAddressName2 = Utilities.RandomResourceName("pip2", 20);
-            string publicIPAddressLeafDNS1 = Utilities.RandomResourceName("pip1", 20);
-            string publicIPAddressLeafDNS2 = Utilities.RandomResourceName("pip2", 20);
-            string networkInterfaceName = Utilities.RandomResourceName("nic", 20);
-            string vmName = Utilities.RandomResourceName("vm", 8);
-            string rgName = Utilities.RandomResourceName("rgNEMP", 24);
-
-            var networkManagementClient = new NetworkManagementClient(SubscriptionId, credential);
-            var virtualNetworks = networkManagementClient.VirtualNetworks;
-            var publicIPAddresses = networkManagementClient.PublicIPAddresses;
-            var networkInterfaces = networkManagementClient.NetworkInterfaces;
-            var computeManagementClient = new ComputeManagementClient(SubscriptionId, credential);
-            var virtualMachines = computeManagementClient.VirtualMachines;
-
+            const string UserName = "tirekicker";
+            const string Password = "RX-78-gp$3";
+            const string publicIPAddressName1 = "pip1";
+            const string publicIPAddressName2 = "pip2";
+            const string publicIPAddressLeafDNS1 = "pipdns1";
+            const string publicIPAddressLeafDNS2 = "pipdns2";
+            const string networkInterfaceName = "nic";
+            const string vmName = "vm";
+            const string rgName = "rgNEMP";
+
+            // create an ArmClient as the entry of all resource management API
+            ArmClient armClient = new ArmClient(new DefaultAzureCredential());
+
+            ResourceGroup resourceGroup = null;
             try
             {
-                await ResourceGroupHelper.CreateOrUpdateResourceGroup(rgName, "eastus");
+                resourceGroup = await armClient.DefaultSubscription.GetResourceGroups().CreateOrUpdate(rgName, new ResourceGroupData("eastus")).WaitForCompletionAsync();
 
                 // Assign a public IP address for a VM during its creation
 
                 // Define a public IP address to be used during VM creation time
 
-                Utilities.Log("Creating a public IP address...");
-
-                var publicIPAddress = new PublicIPAddress
+                Console.WriteLine("Creating a public IP address...");
```

Comment:
Maybe the 'Utilities' is a little bit heavy, but to be constituency with other samples should keep it

Reply:
Unfortunately I cannot use `Utilities` because it's using old incompatible libraries. I could not update `Uitilites` because it's used by Samples which are still on old libraries.